frontend/packages/console-plugin-sdk/src/store.ts

```diff
@@ -0,0 +1,99 @@
+import * as _ from 'lodash';
+import { Map as ImmutableMap } from 'immutable';
+import {
+  Extension,
+  AlwaysOnExtension,
+  ActivePlugin,
+  isFeatureFlag,
+  isModelDefinition,
+} from './typings';
+import { ExtensionRegistry } from './registry';
+
+const isAlwaysOnExtension = (e: Extension): e is AlwaysOnExtension => {
+  return isFeatureFlag(e) || isModelDefinition(e);
```

Comment:
Agreed, will do.

The `isAlwaysOnExtension` implementation follows type declarations - both `FeatureFlag` and `ModelDefinition` extend `AlwaysOnExtension` (instead of `Extension`).

Reply:
I can also move `isAlwaysOnExtension` function (which is technically a [type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)) closer to extension type declarations, i.e. `packages/console-plugin-sdk/src/typings/index.ts` - let me know if I should.